src/routes/committee/rehearsals/+page.svelte

```diff
@@ -61,18 +63,72 @@
 		date = ''
 		startTime = '18:00'
 		endTime = '21:00'
-		openModal = false
+
+		invalidateAll()
+	}
+
+	/** Edit a rehearsal in the database */
+	async function editRehearsal() {
+		// First delete the old document
+		const docRef = doc(db, 'rehearsals', data.rehearsals[selectedRehearsalId].id)
+		await deleteDoc(docRef)
+
+		// Then add a new rehearsal with the new data
+		let dateSplit = date.split('/')
+		let sTimeSplit = startTime.split(':')
+		let eTimeSplit = endTime.split(':')
```

Comment:
I'm a bit out of practice with firebase, but can't you modify a document directly?

Reply:
Yeah you can update a doc

Reply:
https://firebase.google.com/docs/firestore/manage-data/add-data#update-data